src/rules/declaration-no-important/index.js

```diff
@@ -0,0 +1,26 @@
+const ruleName = "declaration-no-important"
+
+export const messages = {
+  unexpected: `Unexpected !important (${ruleName})`,
+}
+
+/**
+ * @param {boolean} options - If true, no !important;
+ *   if false, do nothing
+ */
+export default function declarationNoImportant(options) {
+  return (css, result) => {
+    if (options) {
```

Comment:
I'd suggest swapping this out for an early return to avoid nesting. e.g.

```js
if (!options { return; }
css.eachDecl(..
```

Does that make sense?